app/adapter/graphql/resolver/url_test.go

```diff
@@ -0,0 +1,59 @@
+package resolver
+
+import (
+	"github.com/byliuyang/app/mdtest"
+	"short/app/adapter/graphql/scalar"
+	"short/app/entity"
+	"testing"
+	"time"
+)
+
+func TestURL_Alias(t *testing.T) {
+	urlTest := URL{url: struct {
+		Alias       string
+		OriginalURL string
+		ExpireAt    *time.Time
+		CreatedBy   *entity.User
+		CreatedAt   *time.Time
+		UpdatedAt   *time.Time
+	}{Alias: "TestAlias"}}
+
+	expected := urlTest.url.Alias
+	result := *urlTest.Alias()
+	mdtest.Equal(t, result, expected, "*urlTest.Alias() of '%v' is %v", expected, result)
```

Comment:
```suggestion
	mdtest.Equal(t, got, expected, "Alias() = %v; want %v",  got, expected)
```

Reply:
Please format the rest of like suggested for this one. I'll approve this PR afterwards